lib/Twig/Environment.php

```diff
@@ -213,24 +216,37 @@ public function isStrictVariables()
     }
 
     /**
-     * Gets the cache directory or false if cache is disabled.
+     * Gets the current cache implementation.
      *
-     * @return string|false
+     * @return Twig_CacheInterface|string|false A Twig_CacheInterface implementation,
```

Comment:
Ah yes, didn't see it was to kept bc as it is also set in the `setCache` method. My bad.